server/provider/networkProvider.go

```diff
@@ -388,10 +390,18 @@ func (provider *networkProvider) GetAccount(address string) (*data.AccountModel,
 	}
 
 	options := common.AccountQueryOptions{OnFinalBlock: true}
-	account, err := provider.accountProcessor.GetAccount(address, options)
+	url := common.BuildUrlWithAccountQueryOptions(fmt.Sprintf(urlPathGetAccount, address), options)
+	response := &data.AccountApiResponse{}
+
+	_, err := provider.baseProcessor.CallGetRestEndPoint(provider.observerUrl, url, &response)
```

Comment:
Here, we explicitly perform a GET request against the `baseProcessor`, instead of using proxy's `accountProcessor` (which does not return the original error).

We did not yet remove `accountProcessor` from `networkProvider`, in order to avoid many merge conflicts in context of https://github.com/ElrondNetwork/rosetta/pull/11.

Though, dependency on `accountProcessor` will be removed in the future.